modules/widgets/follow-button.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+add_action( 'widgets_init', 'follow_button_register_widget' );
+function follow_button_register_widget() {
+	register_widget( 'Follow_Button_Widget' );
+}
+
+class Follow_Button_Widget extends WP_Widget {
+
+	public function __construct() {
+		parent::__construct(
+			'follow_button_widget',
+			__( 'Follow Button', 'jetpack' ),
+			array( 'description' => __('Add a WordPress.com follow button to allow people to follow your blog easier', 'jetpack' ) )
+		);
```

Comment:
#3543 and #5440 should give you some examples. [This post](https://make.wordpress.org/core/2016/03/22/implementing-selective-refresh-support-for-widgets/) can also be helpful.